bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx71.html

```diff
@@ -0,0 +1,116 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import monitor_button, fxa_cta_link with context %}
+
+{% add_lang_files "firefox/whatsnew_71" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}What’s new with Firefox - More privacy, more protections.{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('exp_firefox_whatsnew_71') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew71' %}
+{% set _utm_campaign = 'whatsnew71' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success">
+        <p>Congrats! You’re using the latest version of Firefox.</p>
```

Comment:
Question: should strings be wrapped in this template?

Reply:
CRO pages are in English only, so no need to wrap strings for it.